src/lib/components/hamburger-menu/menu.svelte

```diff
@@ -48,7 +48,7 @@
         {/if}
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-6 px-3">
```

Comment:
Instead of having `mt-6` applied here, you can also try using `space-y-6` or `space-y-4` on the `<ul>` element. This is a bit cleaner in my opinion

Reply:
Seems right. Will make the changes